libraries/freertos_plus/aws/ota/src/aws_iot_ota_agent.c

```diff
@@ -1258,7 +1258,9 @@ static OTA_Err_t prvPublishGetStreamMessage( OTA_FileContext_t * C )
                     if( eResult != IOT_MQTT_SUCCESS )
                     {
                         OTA_LOG_L1( "[%s] Failed: %s\r\n", OTA_METHOD_NAME, pcTopicBuffer );
-                        /* Don't return an error. Let max momentum catch it since this may be intermittent. */
+                        /* Don't return an error. Let max momentum catch it since this may be intermittent.
```

Comment:
Can we please move the call to prvStartRequestTimer after if-else block as it is being called in both cases and the if-else blocks will be left for logs only.

Reply:
Updated the change to only restart the timer after the if-else block and added a comment on the logic of why it is always restarted.